### ISSUE TITLE
Windows Defender detected Trojan in latest build

### ISSUE THREAD

**Issue by souvikdas95** [dreamstalker/rehlds#397]
http://teamcity.rehlds.org/repository/download/Rehlds_Publish/6850:id/rehlds-dist-3.0.0.497-dev.zip

-------------------------

Category: Trojan

Description: This program is dangerous and executes commands from an attacker.

Recommended action: Remove this software immediately.

Items: 
containerfile:C:\ProgramData\Microsoft\Windows Defender\Scans\FilesStash\92FFD72B-5A44-FD2E-8084-DCF0F64983AD_1d29b52c94f5813
containerfile:E:\Souvik Das\Downloads\rehlds-dist-3.0.0.497-dev.zip
file:C:\ProgramData\Microsoft\Windows Defender\Scans\FilesStash\92FFD72B-5A44-FD2E-8084-DCF0F64983AD_1d29b52c94f5813->(MPLC)->bin/hlds.exe
file:E:\Souvik Das\Downloads\rehlds-dist-3.0.0.497-dev.zip->bin/hlds.exe
webfile:C:\ProgramData\Microsoft\Windows Defender\Scans\FilesStash\92FFD72B-5A44-FD2E-8084-DCF0F64983AD_1d29b52c94f5813|http://teamcity.rehlds.org/repository/download/Rehlds_Publish/6850:id/rehlds-dist-3.0.0.497-dev.zip|chrome.exe
webfile:E:\Souvik Das\Downloads\rehlds-dist-3.0.0.497-dev.zip|http://teamcity.rehlds.org/repository/download/Rehlds_Publish/6850:id/rehlds-dist-3.0.0.497-dev.zip|chrome.exe

**Comment by In-line** [comment]
It's pure written anti malware software.

**Comment by LevShisterov** [comment]
You can try current version, may be it will be different.
If there would be an easy way to test this with different settings, then may be it could be solved.
But actually I think that only code signing will help for sure.

**Comment by souvikdas95** [comment]
Yeah it's different in the sense that Windows Defender doesn't detect anything as yet but it could have been cooler if VirusTotal could do the same magic. Yes you're right about code signing but the questions are, how to, when and how long will it take for the heuristics to change their decision?

**Comment by In-line** [comment]
Closing as non active.
